cmd/kube-apiserver/app/options/plugins.go

```diff
@@ -83,4 +89,46 @@ func RegisterAllAdmissionPlugins(plugins *admission.Plugins) {
 	setdefault.Register(plugins)
 	resize.Register(plugins)
 	pvcprotection.Register(plugins)
+
+	// Set recommended plugins order
+	recommendedOrder := []string{
+		deny.PluginName,
```

Comment:
Can you add the plugin names as comment behind this? Our package names are not the best to understand out of context.

Reply:
sure